sentinel/src/schedule/outbound.js

```diff
@@ -0,0 +1,232 @@
+const objectPath = require('object-path'),
+      urlJoin = require('url-join'),
```

Comment:
I'm using it to combine the front and back of a URL together cleanly, understanding slashes and so on. I'm not sure if the url library can help here (I did look, and `path` is just for disk paths not for uris):

```js
const base = 'https://example.com/api/v1/'
const path = '/some/action/etc'
whateverYouPropose(base, path) // https://example.com/api/v1/some/action/etc
```

(`resolve` is close but in the example above it would create https://example.com/some/action/etc because of how its rules work)

If you're not comfortable adding a dependency we might get 80% of the way there by just regexing out end slashes of the base and start slashes of the path and then re-adding it back. I just didn't want silly errors in prod where people were getting their slashes wrong.

Reply:
Please no regex. If you've investigated the obvious lib and it doens't work, let's use this other one.